localstack/utils/cli.py

```diff
@@ -61,6 +62,29 @@ def cmd_infra(argv, args):
             start_infra_locally()
 
 
+def cmd_config(argv, args):
+    """
+Usage:
+  localstack config <subcommand> [options]
+
+Commands:
+  config validate       Validate local configurations (e.g. docker-compose)
+
+Options:
+  --file=<>          Run the infrastructure in a Docker container (default)
+  --
```

Comment:
nit: I guess we can remove this line.

Reply:
We are using --file param so it is required.
I have changed the description.